tests/test_mathml2sympy.py

```diff
@@ -58,6 +57,10 @@ def test_eq(self):
         self.assert_equal('<apply><eq/><ci>a</ci><ci>b</ci></apply>',
                           [sympy.Eq(sympy.Symbol('a'), sympy.Symbol('b'))])
```

Comment:
Maybe add an eq with 3 arguments. MathML parses (0 == 0 == 1) as false, (because 0 isn't 1), whereas C will see it as `((0 == 0) == 1) = (1 == 1) = 1`. Python does it like MathML, so guessing sympy does too?

Reply:
Thanks, that's a case I didn't consider. Looks like Sympy relations all handle 2 arguments out of the box but MathML relations are all n-ary. I'll need to implement this.

Reply:
Related: https://github.com/sympy/sympy/issues/6059

Reply:
Fixed in latest commit